crypto/bls_crossBLST_test.go

```diff
@@ -21,8 +18,9 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-	blst "github.com/supranational/blst/bindings/go"
 	"pgregory.net/rapid"
+
+	"github.com/onflow/flow-go/crypto/internal/blst"
```

Comment:
BLST repo also includes Go wrappers. We can't use these Go wrappers directly because:
- not all BLST tools are designed the same way Flow crypto requires them
- many tools needed for Flow crypto aren't exported by BLST (even if they are exported, there will be performance overhead to use them as low level building blocks)

This file tests the overlap of Flow crypto and BLST Go, to make sure there is no divergence.